src/mqtt-broker/src/server/connection.rs

```diff
@@ -23,8 +23,8 @@ static CONNECTION_ID_BUILD: AtomicU64 = AtomicU64::new(1);
 
 #[derive(Clone, PartialEq, PartialOrd)]
 pub enum NetworkConnectionType {
-    TCP,
-    TCPS,
+    Tcp,
+    Tcps,
```

Comment:
Should this be `Tls` instead of `Tcps`? At least `Tls` is much more commonly used in the rust community